Makefile

```diff
@@ -2,6 +2,11 @@ CLANG_FORMAT ?= clang-format
 
 .PHONY: database format clean
 
+env:
```

Comment:
Make this rule .PHONY? Up to you. Note that it won't work if something breaks before the last line. Alternatively, make a directory like `env-in-progress` and `mv` it to a directory called `env`.

Reply:
Done.